app/views/network_router/add_interface_select.html.haml

```diff
@@ -0,0 +1,27 @@
+%form#form_div{:name => "angularForm", 'ng-controller' => "networkRouterFormController"}
+  = render :partial => "layouts/flash_msg"
+  %h3
+    = _('Add Interface to Router')
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Subnet')
+      .col-md-8
+        = select_tag("cloud_subnet_id",
+          options_for_select([["<#{_('Choose')}>", nil]] + @subnet_choices.sort),
+                      "ng-model"                    => "networkRouterModel.cloud_subnet_id",
+                      "required"                    => "",
+                      :miqrequired                  => true,
+                      :checkchange                  => true,
+                      "selectpicker-for-select-tag" => "")
+  %table{:width => '100%'}
+    %tr
+      %td{:align => 'right'}
+        #buttons_on
+          = render :partial => "layouts/angular/x_custom_form_buttons_angular",
+                   :locals  => {:button_label           => N_("Add"),
```

Comment:
Sure. I just noticed that you made this same change last month to the code that I modeled this from -- I'll make the same change here.